flaml/automl/automl.py

```diff
@@ -331,7 +351,7 @@ def _prepare_sample_train_data(self, sample_size):
         return sampled_X_train, sampled_y_train, sampled_weight, groups
 
     @staticmethod
-    def _compute_with_config_base(config_w_resource, state, estimator):
+    def _compute_with_config_base(config_w_resource, state, estimator, is_report=True):
```

Comment:
I don't think `is_report` makes sense here. It's not that something is or isn't a report, but rather that if we're using ray then we need to report the score back to tune. Might be better just to pass `use_ray`

Reply:
Maybe we will implement async report for Spark too. In that case, would `is_report` make more sense? Plus, `use_ray` kinds of limiting things to `ray` or `not ray`, but is the fact here more about `report` or `not report`?